### PR TITLE
exiflooter: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Powerful tool that finds geolocation on all image urls and directories also integrates with OpenStreetMap.

Source: https://github.com/aydinnyunus/exifLooter

### CODE REVIEW

File: packages/exiflooter/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+license=('Apache-2.0')
+depends=('perl-image-exiftool')
+makedepends=('git' 'go')
+source=("git+https://github.com/aydinnyunus/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  # use, if no version string provided neither in sources nor in git describe:
```
Comment (noraj) [inline packages/exiflooter/PKGBUILD:20]: remove comments

File: packages/exiflooter/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+
+  # if tag exists, use this
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```
Comment (noraj) [inline packages/exiflooter/PKGBUILD:24]: remove the unneeded line, you already picked the git count method

File: packages/exiflooter/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# See COPYING for license details.
+
+pkgname=exiflooter
+pkgver=33.a92e697
+pkgrel=1
+pkgdesc='Find geolocation on all image urls and directories also integrates with OpenStreetMap.'
+arch=('any')
+groups=('blackarch' 'blackarch-forensic')
+url='https://github.com/aydinnyunus/exifLooter'
+license=('Apache-2.0')
```
Comment (noraj) [inline packages/exiflooter/PKGBUILD:11]:
```suggestion
license=('Apache')
```

cf. https://wiki.archlinux.org/title/PKGBUILD#license

File: packages/exiflooter/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+
+  # if tag exists, use this
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+  cd $pkgname
+
+  # If you encounter the error: go mod download: no modules specified (see 'go help mod download')
```
Comment (noraj) [inline packages/exiflooter/PKGBUILD:30]: remove the template comments

File: packages/exiflooter/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+package() {
+  cd $pkgname
+
+  install -Dm 755 $pkgname "$pkgdir/usr/bin/$pkgname"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  # Only install license if required according to
+  # https://wiki.archlinux.org/index.php/PKGBUILD#license
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/exiflooter/PKGBUILD:52]: The comment above was exactly explaining not to include the license in this case

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/exiflooter/PKGBUILD (hunk 2 above)
Comment (D3vil0p3r) [reply packages/exiflooter/PKGBUILD]: Removed

File: packages/exiflooter/PKGBUILD (hunk 1 above)
Comment (D3vil0p3r) [reply packages/exiflooter/PKGBUILD]: Removed

File: packages/exiflooter/PKGBUILD (hunk 4 above)
Comment (D3vil0p3r) [reply packages/exiflooter/PKGBUILD]: Removed

File: packages/exiflooter/PKGBUILD (hunk 5 above)
Comment (D3vil0p3r) [reply packages/exiflooter/PKGBUILD]: Removed

**Review by noraj — APPROVED** [review]
Tested, LGTM.

**Pull request merged**